checker/src/main/java/org/checkerframework/checker/nullness/KeyForValue.java

```diff
@@ -84,8 +84,10 @@ public KeyForValue(
   }
 
   @Override
-  public KeyForValue leastUpperBound(@Nullable KeyForValue other) {
-    KeyForValue lub = super.leastUpperBound(other);
+  protected KeyForValue upperBound(
+      @Nullable KeyForValue other, TypeMirror upperBoundTypeMirror, boolean shouldWiden) {
+    KeyForValue lub = super.upperBound(other, upperBoundTypeMirror, shouldWiden);
```

Comment:
This variable name suggests that `upperBound()` actually returns the least upper bound.  Why is `least` not in its name?

OK, I see now.  I wonder if its name should be `leastUpperBoundOrWiden()`, for clarity.